tests/integration/shell/key.py

```diff
@@ -141,6 +172,34 @@ def test_list_acc(self):
         expect = ['Accepted Keys:', 'minion', 'sub_minion']
         self.assertEqual(data, expect)
 
+    def test_list_acc_eauth(self):
+        '''
+        test salt-key -l with eauth
+        '''
+        self._add_user()
+        data = self.run_key('-l acc --eauth pam --username {0} --password {1}'.format(USERA, USERA_PWD))
+        expect = ['Accepted Keys:', 'minion', 'sub_minion']
+        self.assertEqual(data, expect)
+        self._remove_user()
+
+    def test_list_acc_eauth_bad_creds(self):
+        '''
+        test salt-key -l with eauth and bad creds
+        '''
+        self._add_user()
+        data = self.run_key('-l acc --eauth pam --username {0} --password wrongpassword'.format(USERA))
+        expect = []
```

Comment:
I've compared the behavior to `salt` and `salt-run` and updated it here. So please change the test to expect `Authentication failure of type "eauth" occurred for user <username>.` message.
Thank you for catch @Ch3LL !